web/themes/joinup/templates/content/page-title--node--news.html.twig

```diff
@@ -15,16 +15,16 @@
 {{ title_prefix }}
 {% if title %}
   <div class="page__title-wrapper">
-      <h1 class="page__title">{{ title }}</h1>
+    <div class="details">
+      <div class="details__element details__element-headline">
+        <div class="value gray">{{ title }}</div>
```

Comment:
Hi @tom-waat I was reviewing latest changes and I think it's good to go comparing with Andrea's requirements here.
However, tests are failing because the headings are having changed file structures like following:

Success, as expected:
![passing](https://user-images.githubusercontent.com/1923476/29575066-ad571832-8763-11e7-901f-b2a28b1bbca4.png)

Failing tests, because the text is not in h1, h2, etc. tag:
![failling](https://user-images.githubusercontent.com/1923476/29575077-b8dd3448-8763-11e7-9a90-740ad8dfbf46.png)

Do you think the markup should be updated to match the test assertions or rather the other way around?

Apart from these changes we need to handle, I'd like to merge this one.